src/selic-entries.ts

```diff
@@ -1,15 +1,25 @@
-import { format, compareAsc } from "date-fns"
+import { format, compareAsc } from "date-fns";
 
 export async function selicEntries(initialDate: Date, finalDate: Date) {
```

Comment:
Ticket #7 asks for both parameters be optional, and that's not possible with this definition, nor the implementation would allow it.

Reply:
Is it better to have the validation of the parameters inside the method instead of being required arguments? Is it because of JavaScript?